chef/cookbooks/ceilometer/recipes/server.rb

```diff
@@ -325,23 +325,6 @@
   action :add_endpoint_template
 end
 
-# Cronjob to clean the database. The effectiveness of this depends on
-# the database in use.
-# http://docs.openstack.org/admin-guide-cloud/content/section_telemetry-storing-data.html
-if node[:ceilometer][:database][:time_to_live] > 0
-  cookbook_file 'cronjob-ceilometer-expirer' do
-    path '/etc/cron.daily/crowbar-ceilometer-expirer'
-    owner 'root'
-    group 'root'
-    mode '0755'
-    action :create
-  end
-else
-  file '/etc/cron.daily/crowbar-ceilometer-expirer' do
-    action :delete
-  end
```

Comment:
You should probably have kept that, as the cron job will be in stoney/icehouse and so we need to remove it in tex/juno.